projects/dendrites/permutedMNIST/experiments/__init__.py

```diff
@@ -36,6 +38,10 @@
 # Collect all configurations
 CONFIGS = dict()
 CONFIGS.update(BASE)
+CONFIGS.update(CENTROID)
+CONFIGS.update(HYPERPARAMETERSEARCH)
+CONFIGS.update(SP_PROTO)
+CONFIGS.update(SP_CONTEXT)
```

Comment:
Here, you only need `CONFIGS.update(HYPERPARAMETERSEARCH)` and should remove the other 3 lines, because they already exist in this file!